runtime/plugins/comment/comment.lua

```diff
@@ -61,17 +62,20 @@ ft["zig"] = "// %s"
 ft["zscript"] = "// %s"
 ft["zsh"] = "# %s"
 
-local last_ft
-
 function updateCommentType(buf)
-    if buf.Settings["commenttype"] == nil or (last_ft ~= buf.Settings["filetype"] and last_ft ~= nil) then
+    if buf.Settings["commenttype"] ~= nil then
+        micro.InfoBar():Error("\"commenttype\" option has been updated to \"comment.type\"",
+                              "instead, please update accordingly")
```

Comment:
I think this wording is harder to misinterpret:

```suggestion
        micro.InfoBar():Error("\"commenttype\" option has been renamed to \"comment.type\"",
                              ", please update your configuration")
```